cartoframes/data/observatory/catalog/geography.py

```diff
@@ -216,6 +219,9 @@ def download(self, credentials=None):
         :raises CartoException: If you have not a valid license for the dataset being downloaded.
         :raises ValueError: If the credentials argument is not valud.
         """
+        if not self._is_subscribed(credentials):
+            raise CartoException('You are not subscribed to this Geography yet. Please, use the subscribe method '
```

Comment:
Same as before with the `CartoException`

Reply:
Same response :)